recipes/edward/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "edward" %}
+{% set version = "1.3.3" %}
+{% set sha256 = "104d58321c5040235b3039ae3215c3c7881073e6aa88bb0b8ca1141ca87c4891" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
```

Comment:
I don't think `numpy` or `six` is required for the `build` as the recipe is using `--single-version-externally-managed`